content/2_Bronze/Ad_Hoc.mdx

```diff
@@ -35,6 +36,286 @@ approaching problems that appear to be ad hoc.
 In the end, the best way to get better at ad hoc problems (or any type of
 problem) is to do a lot of them.
 
+### Example - Milking Order
+
+<FocusProblem problem="tutorial" />
+
+Don't be afraid to give up on some approach if you aren't making progress!
+
+<Spoiler title="Hint 1">
+Let's try placing cow $1$ at a specific position.
+
+How might we go about checking to see if we'll end up with a valid ordering?
+</Spoiler>
+
+<Spoiler title="Solution 1">
```

Comment:
in modules, we're putting all solutions out of spoiler blocks
hints can still be in them tho